mod/photos.php

```diff
@@ -914,17 +914,15 @@ function photos_content(App $a)
 			'$submit' => DI::l10n()->t('Submit'),
 		]);
 
-		// Determine which setting actually limits  upload size of images: limit of Friendica ('maximagesize') or upload_max_filesize
 		$maximagesize_Mbytes = 0;
 		// Get the relevant size limits for uploads. Abbreviated var names: MaxImageSize -> mis; upload_max_filesize -> umf
 		$mis_bytes = DI::config()->get('system', 'maximagesize');
-		$umf_bytes = Strings::getBytesFromShorthand(get_cfg_var('upload_max_filesize'));
+		$umf_bytes = Strings::getBytesFromShorthand(ini_get('upload_max_filesize'));
+
+		// When PHP is configured with upload_max_filesize less than maximagesize provide this lower limit.
+		(($umf_bytes < $mis_bytes) || ! is_numeric($mis_bytes)) ?
+			($maximagesize_Mbytes = ($umf_bytes / (10 ** 6))) : ($maximagesize_Mbytes = ($mis_bytes / (10 ** 6)));
```

Comment:
Please flip the condition. Since it's lazily evaluated, if it isn't numeric, it won't be compared at all. While you're at it, you can also remove some code duplication in your structure.
```suggestion
		$maximagesize_Mbytes = (is_numeric($mis_bytes) && ($mis_bytes < $umf_bytes) ? $mis_bytes : $umf_bytes) / (10 ** 6);
```

Reply:
thx! :)

Reply:
changed it but also replace the unnecessary mulitplication 10 ** 6 with 1048576